docs/products/clickhouse/howto/clickhouse-query-cache.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Use query cache in Aiven for ClickHouse®
+sidebar_label: Use query cache
+---
+
+Aiven for ClickHouse® provides a query cache mechanism that helps improve query performance
+by caching query results.
+
+## How it works
+
+When the Aiven for
+[ClickHouse query cache](https://clickhouse.com/docs/en/operations/query-cache) is enabled,
+multiple identical `SELECT` queries run simultaneously are computed only once,
+and further executions of the same query are served directly from the cache. If the result
+of a SELECT query changes, the database invalidates and discards cached query results.
+Operations that change the data are `INSERT`, `UPDATE`, or `DELETE`.
+
+:::important
+By default, the Aiven for ClickHouse query cache is disabled for all `SELECT` queries.
+:::
+
+## Why use it
+
+Using query cache in your Aiven for ClickHouse services can help reduce latency and
+resource consumption.
+
+Key use case for the Aiven for ClickHouse query cache are the following:
+
+- Performance enhancement: Reducing latency and load for frequently executed queries
+- Analytical workloads: Using complex repetitive queries that involve high data aggregation
+- Data pipelines: [Using materialized views](/docs/products/clickhouse/concepts/federated-queries)
```

Comment:
I do not understand how materialized views are related to the Query Cache. Looks like it is actually another way to handle the problem